frontend/src/modules/Survey/Components/Step3.tsx

```diff
@@ -1,53 +1,75 @@
 import React from 'react'
 import { colors, h2, body } from '@core'
+import { YearField } from '@core'
 import {
   StyledContainer,
   StyledText,
   StyledCalendarWrapper,
   StyledTextField,
-  StyledCalendarLabel,
+  StyledErrorWrapper,
+  StyledErrorIcon,
+  StyledErrorText,
 } from 'Survey/Styles/Step3.style'
 import { createStyles, makeStyles, Theme } from '@material-ui/core/styles'
+import { StepProps } from 'Survey/Types/StepProps'
+import {
+  getYoungestGradYear,
+  getOldestGradYear,
+} from 'Survey/Components/FuncsAndConsts/SurveyFunctions'
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    container: {
-      display: 'flex',
-      flexWrap: 'wrap',
-    },
-    textField: {
-      marginLeft: theme.spacing(1),
-      marginRight: theme.spacing(1),
-      width: 200,
+export const Step3 = ({ showError, setAnswer, currentAnswer }: StepProps) => {
+  const textContainerStyle = {
+    margin: '0.75rem 0',
+    root: {
+      background: 'white',
+      borderRadius: 3,
+      border: 0,
+      height: 48,
+      padding: '0 30px',
+      '& .MuiInput-underline:before': {
+        borderBottomColor: colors.lightviolet, // Semi-transparent underline
+      },
+      '& .MuiInput-underline:hover:before': {
+        borderBottomColor: colors.lightviolet, // Solid underline on hover
+      },
+      '& .MuiInput-underline:after': {
+        borderBottomColor: colors.lightviolet, // Solid underline on focus
+      },
+      '& .MuiInputBase-root': {
+        color: colors.lightviolet,
+      },
+      '& .MuiInputBase-root:focus': {
+        color: colors.lightviolet,
+      },
     },
-  })
-)
+  }
 
-export const Step3 = () => {
-  const defaultGradDate = '2024/05'
-  const [gradDate, setGradDate] = React.useState<string>(defaultGradDate)
-  const classes = useStyles()
+  const textInputStyle = {
+    fontWeight: '800',
+    color: '#E5CEFA',
```

Comment:
let's import 'colors' from '@core' for this. I think i defined a 'violet' there.